src/components/options.tsx

```diff
@@ -67,7 +67,23 @@ const MarkdownPanel: React.FC = () => {
 };
 
 const CssPanel: React.FC = () => {
-	return null;
+	const explorer = useExplorer();
+	const { cssOptions, setCssOptions } = explorer;
+	const { tolerant } = cssOptions;
+	return (
+		<>
```

Comment:
These are react Fragments and these aren't needed when the component returns one element. In this case it is returning a single `div` so you can remove it